### PR TITLE
chore: add density slider to the development site

### LINKED ISSUE #907: task: add density slider to development site

### PR DESCRIPTION
**Pull request by khamudom**
density slider is added and hooked up to work. 

closes #907

### CODE REVIEW

**Comment by janechu** [comment]
> density slider is added and hooked up to work #907

In order to close a task on github please use the correct formatting for this message, see: https://help.github.com/articles/closing-issues-using-keywords/

File: packages/fast-components-react-msft/app/app.tsx [hunk 1]
```diff
@@ -165,6 +185,12 @@ export default class App extends React.Component<{}, IAppState> {
         );
     }
 
+    private handleDensityUpdate = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        const value: number = parseInt(e.target.value, 10);
+        this.setState({
+            density: value
```
Comment (janechu) [inline packages/fast-components-react-msft/app/app.tsx]: you can remove adding the const and just put `density: parseInt(e.target.value)`

File: packages/fast-components-react-msft/app/app.tsx [hunk 2]
```diff
@@ -47,6 +48,7 @@ enum Theme {
 export interface IAppState extends IColorConfig {
     theme: Theme;
     direction: Direction;
+    density: number;
```
Comment (janechu) [inline packages/fast-components-react-msft/app/app.tsx]: please add this to the documentation for this package via the `README.md`.

Comment (chrisdholt) [reply packages/fast-components-react-msft/app/app.tsx]: @janechu this is in MSFT only; we aren't documenting the others in the readme from what I see. Should we? This is just a configuration of the dev site with custom slider for density.

File: packages/fast-components-react-msft/app/app.tsx (hunk 1 above)
Comment (khamudom) [reply packages/fast-components-react-msft/app/app.tsx]: updated

**Comment by nicholasrice** [comment]
Can you rename this PR to "chore(fast-components-react-msft): adding density slider to the development site"? A "feat" tells the changelog generator that this package needs  minor bump because of this change, however this change has no effect on distributed package.

File: packages/fast-components-react-msft/app/app.tsx [hunk 3]
```diff
@@ -100,12 +103,28 @@ export default class App extends React.Component<{}, IAppState> {
...
+                            style={{marginRight: "8px"}}
+                        >
+                            density
+                        </Label>
+                        <input
+                            type="range"
+                            name="density"
+                            defaultValue="0"
+                            min="-3"
+                            max="3"
```
Comment (janechu) [inline packages/fast-components-react-msft/app/app.tsx]: @nicholasrice might be able to give better direction on this but I believe the min should be `0.4` for now. We need to work on the density story a little to find out the proper range.

**Pull request merged**